content/03.methods.md

```diff
@@ -144,6 +144,20 @@ Theta2 purity was added as an optional parameter to CNVkit to adjust copy number
 CNVkit was run on human genome reference hg38 using the optional parameters of Theta2 purity and BAF adjustment for tumor-normal pairs.
 We used GISTIC [@doi:10.1186/gb-2011-12-4-r41] v.2.0.23 on the CNVkit and the consensus CNV segmentation files to generate gene-level copy number abundance (Log R Ratio) as well as chromosomal arm copy number alterations using the parameters specified in the [OpenPBTA Analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/run-gistic/scripts/run-gistic-openpbta.sh).
 
+#### Focal Copy Number Calling
+
+We added the ploidy inferred via ControlFreeC to the CNVkit and consensus CNV segmentation files and used the ploidy and copy number values to define a status column with gain and loss values broadly.
+We used bedtools coverage [@url:https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html] to add cytoband status using the UCSC cytoband file [@url:http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz].
+The output status call fractions were used to define dominant status at the cytoband-level.
+The weighted means of each status call fraction and band length were calculated and used to define the dominant status at the chromosome arm-level.
+
+A status is considered dominant if the more than half of the region was callable and the status call fraction was greater than 0.9 for that region.
+The 0.9 threshold was chosen to ensure that the dominant status fraction call is indeed significantly greater than the remaining status fraction calls in a region.
```

Comment:
```suggestion

```
I don't know if you need this sentence. At the very least I would remove the phrase "indeed significantly greater."